src/aap_eda/core/utils/decision_environment.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from aap_eda.core.enums import ImagePullPolicy
+
+
+def convert_pull_policy_to_frontend(pull_policy: str) -> str:
+    """Convert internal pull policy enum to frontend-friendly string."""
+    policy_mapping = {
```

Comment:
Why do we need to define a set of frontend values? The UI has freedom convert the enums to any string. How does the API doc look like? How does other API endpoints handle enum?

Reply:
> Right now for podman we are implementing the policy in our side but as you say in comment, once the pypodman client support it, we are going to pass the values for podman, so you would use "convert_pull_policy_to_frontend" for it?
> 
All these changes are temporary. When podman 5.6 is available, they will not be needed. We simply pass the pull_policy to podman-py APIs, like what we do now for kubernetes.

Reply:
> 
> All these changes are temporary. When podman 5.6 is available, they will not be needed. We simply pass the pull_policy to podman-py APIs, like what we do now for kubernetes.

exactly, and then you would need to convert them as for the UI. Since podman values are what controller and ui uses, makes sense to use those values by default, and convert only k8s.